src/hyp3_isce2/insar_stripmap.py

```diff
@@ -24,31 +32,86 @@
     os.environ['PATH'] = str(ISCE_APPLICATIONS) + os.pathsep + os.environ['PATH']
 
 
-def insar_stripmap(reference_scene: str, secondary_scene: str) -> Path:
+# TODO update docstring description
+def insar_stripmap(user: str, password: str, reference_scene: str, secondary_scene: str) -> Path:
     """Create an interferogram
 
     This is a placeholder function. It will be replaced with your actual scientific workflow.
 
     Args:
+        user: Earthdata username
+        password: Earthdata password
         reference_scene: Reference scene name
         secondary_scene: Secondary scene name
 
     Returns:
         Path to the output files
     """
+    session = asf.ASFSession().auth_with_creds(user, password)
+
+    results = asf.granule_search([reference_scene, secondary_scene])
```

Comment:
We can automatically filter the search results to `L1.0` scenes. Also, I assume we expect to always receive exactly two `L1.0` products in the search results. Let me know if this is not the case.
```suggestion
    results = asf_search.search(
        granule_list=[reference_scene, secondary_scene],
        processingLevel=asf_search.L1_0,
    )
    assert len(results) == 2
```